src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     internal sealed partial class SyntaxTreeIndex : IObjectWritable
     {
         private const string PersistenceName = "<SyntaxTreeIndex>";
-        private const string SerializationFormat = "12";
+        private const string SerializationFormat = "15";
```

Comment:
Specifically not incrementing by one.  I think there's another PR going in now that is changing this value, and i want them to conflict.

Reply:
If you mean https://github.com/dotnet/roslyn/pull/28230, it's been merged.